scripts/run_nodes.sh

```diff
@@ -1,123 +1,289 @@
-#!/bin/bash
+#!/usr/bin/env bash
+
+# This script runs locally aleph-node consensus, ie set of aleph-node processes that are either
+# validators (create blocks) or RPC nodes (read only nodes). By default, such consensus consist of
+# one RPC node that is a bootnode, and 6 validator nodes.
```

Comment:
Why do we increase validators to 6? I think in most scenarios, 4 is sufficient, no?

Reply:
To make committee rotation feature actually applied, as default `CommitteeSize` is 4 reserved and 0 non-reserved